tests/_support/Log/Handlers/TestHandler.php

```diff
@@ -19,6 +19,8 @@ class TestHandler extends \CodeIgniter\Log\Handlers\FileHandler
 
     /**
      * Where would the log be written?
+     *
+     * @param array $config
      */
     //--------------------------------------------------------------------
```

Comment:
Not directly related to this PR, but I recall there was a discussion about these dividers.
Not sure how it concluded, but looks weird here.

Reply:
Lonnie wants to remove those also. Exception is when the dividers are used as a form of header
```
//---------------------------
// Some comment
//---------------------------
```

I have a custom fixer that removes these dividers. Just need some refining to skip header usages.